com.unity.ml-agents.extensions/Documentation~/InputActuatorComponent.md

```diff
@@ -0,0 +1,42 @@
+# Integration of the Input System Package with ML-Agents
+
+## Overview
+One area we are always trying to improve is getting developers up and running with ML-Agents.  With this in mind,
+we have implemented an `InputActuatorComponent` that sets up an action space for your `Agent` based on
+an `InputActionAsset` that is referenced by the `IInputActionAssetProvider` interface, or the `PlayerInput` component
```

Comment:
In the pushblock integration, the Pushbloc Agent implements a IInputActionAssetProvider but there is no information on what it is an on how it is used.

Reply:
good point.

Reply:
added info in the doc in the extensions package.